src/System/Components/RouterLink.tsx

```diff
@@ -84,11 +88,31 @@ export const RouterLink: React.FC<React.PropsWithChildren<React.PropsWithChildre
       }
     }
 
+    // Inject a query param to indicate that the link was successfully pre-fetched.
+    const updatedHrefWithPrefetchQueryParam = (
```

Comment:
This whole method can probably be cleaned up/improved. Want to append `is_prefetched=true` to the `to`, but also account for if `to` itself already has query params (prob an edge case).